docs/content/setup/getting-started.md

```diff
@@ -0,0 +1,10 @@
+# Getting started
```

Comment:
Should this really be called 'Getting started'?

Reply:
Yes, because this document should be linked from the homepage. It's in the setup page space, because it's more likely to be found there.